README.md

```diff
@@ -59,14 +60,48 @@ To install, simply download the modulr.py script into your project's root direct
 Run the script with Python: `python modulr.py`.
 
 Modulr requires Python >= 3.12 because of the use of `Path.walk`. I plan to eventually
-package Modulr as a standalone commandline app, but for now, you will need Python 3.12
+package Modulr as a standalone commandline app; but for now, you will need Python 3.12
 installed in order to run the script.
 
 If your system doesn't come with 3.12, the easiest way to get a different version
 installed is with either
 [Hatch](https://hatch.pypa.io/latest/install/) or
 [Pyenv](https://github.com/pyenv/pyenv?tab=readme-ov-file#installation).
 
+## Project Scope & Limitations
+
+After careful consideration, there are some features that this project does not intend to
+support. Justification for each is included beneath.
+
+1. Including a component multiple times within the same page.
+
+Modulr endeavors to improve maintainability by allowing you to develop common page
+components once and copy-paste them to your desired pages before release. Many of these
+components will likely involve some level of DOM manipulation, so including these multiple
+times on the same page will inevitably lead to selection collisions. Any suitable solution
+to this challenge will result in a leaky abstraction, which runs counter to Modulr's goal
+of improving maintainability, so this project is built with the philosophy that Modulr
+components are single-use within a page.
+
+Examples of components that may make for good Modulr components include navigation bars,
+page headers, and site footers.
+
+2. Nesting Modulr components within each other.
+
+Rather than the SPA and Native Web Component interpretation of what a "component" is,
+Modulr components are intended to be factored-out, shared sections of web pages. Aside from
+this being out-of-scope for this project, Modulr aims to help you reduce boilerplate. Nested
+components add more overhead to your code, and while how/when to break up components is (to
+some extent) a stylistic choice within some front-end technologies, this process easily devolves
+into code that is difficult to compose and reason about. Not supporting this feature should help
+encourage components that are easier to maintain.
+
+3. Passing properties (AKA "props") to components.
+
+This runs very closely with scope item number two. Due to many of the same reasons why
+nesting components adds complexity to projects, passing properties to components is another
+major source of complexity in your code that this project avoids.
+
```

Comment:
I think we can just remove this section all-together. Props are a product of SPA frameworks, and this project only uses the core language features and web standards. I don't think we need to mention not supporting things like "Props". Also, I never actually considered it since I personally think that Props are an anti-pattern even in SPAs, so I don't think it fits into this section about features that were explored and axed.

It's kind of like "state management". That's a problem that SPAs created that doesn't exist in standard tech, so there's no need to mention it in the context of this project.

If a user opens an issue about supporting Props, I'd probably just respond with "this isn't React", so I'm not really worried about this specifically.

I guess I just don't really want to explain how this project works in terms of some framework or other tool. I understand that it can make it easier for some developers to pick up on the concepts, but it's kind of like learning JS concepts through React or Vue concepts. It's kind of backwards, if that makes sense.